kuma/users/tests/test_templates.py

```diff
@@ -505,27 +541,24 @@ def test_not_banned_user_one_click_page_template_ban_button(self):
         eq_(ban_button.text(), "Ban User for Spam")
         eq_(len(ban_other_reasons), 0)
 
```

Comment:
While this comment goes with the next section, and is misleading. The revisions are created by ``self.testuser``, submitted as spam by ``self.testuser2``, and then ``self.testuser2`` is loaded for banning. Milliseconds from a spam moderator to getting banned!

This repeated test section seems redundant - it seems to be testing the exact same thing as the previous request / response tests, except also creating some irrelevant revisions for another user. Unless I'm missing something (code coverage?) it seems that the 4-line comment and following code block can be dropped.